workflow/gs-stateful-processes.adoc

```diff
@@ -181,7 +181,155 @@ This command should compile and package the process service into a Quarkus execu
 
 The first execution downloads any artifacts not yet available on the development environment
 
-==  Configuring the process runtime database
+== Enabling JDBC Persistence for {BAMOE} stateful processes
+
+{BAMOE} process service and all the {BAMOE} subsystems (Data Index, Data Audit, Job Service and User Tasks) use JDBC to persist
+ in relational databases. The supported databases are PostgreSQL and Microsoft SQL Server for production environments
+and H2 for development.
```

Comment:
IMO statements on supported environments should be kept outside of the documentation, eg. there will be a separate doc for each release anyway.

I think for the rest of this chapter it is still valid to have the instructions for the different DBs, but I would rather not put this "support" statement into the docs.